releng/org.eclipse.tracecompass.testing/feature.xml

```diff
@@ -207,4 +207,8 @@
          id="org.eclipse.tracecompass.slf4j.binding.simple.properties"
          version="0.0.0"/>
 
+   <plugin
+         id="org.eclipse.tracecompass.datastore.core.tests"
+         version="0.0.0"/>
+
 </feature>
```

Comment:
Other plug-ins that could be considered to be added, here or in another commit if applicable:
o.e.tc.lami.ui.swtbot.tests (fragment)
o.e.tc.analysis.profiling.ui.swtbot.tests
o.e.tc.integration.core.tests
o.e.tc.jsontrace.core.tests
o.e.tc.rcp.ui.tests